python/src/aac/plugins/gen_protobuf/gen_protobuf_impl.py

```diff
@@ -53,7 +50,7 @@ def generate_protobuf_messages(architecture_file: str, output_directory: str) ->
 
     # Validate the source AaC file and its contents
     with validated_source(architecture_file) as validation_result:
-        loaded_templates = load_default_templates("gen_protobuf")
+        loaded_templates = load_templates(__package__, ".")
```

Comment:
What's the motivation to not giving Gen-Protobuf a templates directory? Is it because it only has the one template?